pkg/controller/disruption/disruption.go

```diff
@@ -705,39 +721,36 @@ func (dc *DisruptionController) trySync(ctx context.Context, pdb *policy.PodDisr
 	}
 	// We have unmamanged pods, instead of erroring and hotlooping in disruption controller, log and continue.
 	if len(unmanagedPods) > 0 {
-		klog.Warningf("found unmanaged pods associated with this PDB: %v", unmanagedPods)
-		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "UnmanagedPods", "Pods selected by this PodDisruptionBudget (selector: %v) were found "+
```

Comment:
Here's the reason for the test failure: this shouldn't get removed.

Reply:
Good catch! I have reverted it.